pkg/apis/push/v1alpha1/androidvariant_types.go

```diff
@@ -35,8 +35,9 @@ type AndroidVariantSpec struct {
 // AndroidVariantStatus defines the observed state of AndroidVariant
 // +k8s:openapi-gen=true
 type AndroidVariantStatus struct {
-	Ready     bool   `json:"ready"`
-	VariantId string `json:"variantId,omitempty"`
+	Ready         bool   `json:"ready"`
+	VariantId     string `json:"variantId,omitempty"`
+	VariantSecret string `json:"variantSecret,omitempty"`
```

Comment:
Given that this is called `Secret` in UPS, rather than `VariantSecret`, I think we should have it as `Secret` here too.

Reply:
I don't mind changing it, but it's variant secret in mdc and also in ups ui